src/Tribe/Aggregator/Page.php

```diff
@@ -330,6 +329,18 @@ public function template( $name, $data = array(), $echo = true ) {
 		 * @param array  $data     The Data that will be used on this template
 		 */
 		$file = apply_filters( 'tribe_aggregator_template_file', $file, $name, $data );
+		/**
+		 * Crate a new filter using the file name as part of the filter to remove logic from callbacks.
+		 *
+		 * @since TBD
+		 *
+		 * @param string       $file Complete path to include the PHP File.
+		 * @param string|array $name Template name.
+		 * @param array        $data The Data that will be used on this template.
+		 *
+		 * @return string Complete path to the file to be included.
+		 */
+		$file = apply_filters( "tribe_aggregator_template_file/{$file_name}", $file, $name, $data );
```

Comment:
:warning: Words in hook names should be separated using underscores. Expected: "tribe_aggregator_template_file_{$file_name}", but found: "tribe_aggregator_template_file/{$file_name}".

<sub>hash: 07f84e9f5532c72ed26e8de6bb80f1ad</sub>